jboss/container/wildfly/launch/port-offset/added/launch/ports.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# only processes a single environment as the placeholder is not preserved
+
+function prepareEnv() {
+  unset PORT_OFFSET
+}
+
+function configure() {
+  configure_port_offset
+}
+
+function configureEnv() {
+  configure
+}
+
+function configure_port_offset() {
+  if [ -n "${PORT_OFFSET}" ]; then
+    # There is no marker for this one, so look for 'port-offset="0"' to decide on xml vs cli
+    local mode
+    getConfigurationMode "port-offset=\"0\"" "mode"
```

Comment:
@kabir , do we need the sed part? Could CLI be used in all cases?

Reply:
@jfdenise Not sure, we do have an 'xml' only setting for the configuration mode though. Although it is doubtful if anyone will use that. I don't think it matters too much though?

Reply:
Yes, you are right, the xml only. I forgot about that.